build/di.cfg

```diff
@@ -30,6 +30,11 @@ OpenLayers/Format/CQL.js
 OpenLayers/Format/GeoJSON.js
 OpenLayers/Format/XML.js
 OpenLayers/Format/WMSCapabilities/v1_1_1.js
+OpenLayers/Format/WFSCapabilities.js
+OpenLayers/Format/WFSCapabilities/v1.js
+OpenLayers/Format/WFSCapabilities/v1_0_0.js
+OpenLayers/Format/WFSCapabilities/v1_1_0.js
+OpenLayers/Format/WFSDescribeFeatureType.js
```

Comment:
Why do we have to include multiple versions of `WFSCapabilities`?  (I''m sorry, I don't understand this config file format)